### PR TITLE
backend support for offboard/set_attitude

### PR DESCRIPTION
**Pull request by a6a3uh**
`set_attitude` method was added recently to `offboard` plugin and it is not supported in resent `offboard` grpc backend introduced with great [pull request](https://github.com/Dronecode/DronecodeSDK/pull/735) So this adds set_attitude support.

### CODE REVIEW

**Comment by a6a3uh** [comment]
The corresponding Droncode-Proto PR is [here](https://github.com/Dronecode/DronecodeSDK-Proto/pull/83)

**Comment by a6a3uh** [comment]
Corresponding DronecodSDK-Python [PR](https://github.com/Dronecode/DronecodeSDK-Python/pull/64)

File: backend/test/offboard_service_impl_test.cpp [hunk 1]
```diff
@@ -158,19 +183,47 @@ TEST_F(OffboardServiceImplTest, setAttitudeRateDoesNotFailWithNullResponse)
...
-    rpc_attitude_rate->set_yaw_deg_s(ARBITRARY_YAW);
+    rpc_attitude_rate->set_roll_deg_s(ARBITRARY_ROLL_RATE);
+    rpc_attitude_rate->set_pitch_deg_s(ARBITRARY_PITCH_RATE);
+    rpc_attitude_rate->set_yaw_deg_s(ARBITRARY_YAW_RATE);
     rpc_attitude_rate->set_thrust_value(ARBITRARY_THRUST);
 
     return rpc_attitude_rate;
 }
 
+TEST_F(OffboardServiceImplTest, setAttitudeCorrectrly)
```
Comment (JonasVautherin) [inline backend/test/offboard_service_impl_test.cpp]:
```suggestion
TEST_F(OffboardServiceImplTest, setsAttitudeCorrectly)
```

**Comment by JonasVautherin** [comment]
Oh, and you should fix the indentation with `$ make fix_style`, too.

**Pull request merged**